### PR TITLE
analisys summary with new styles.

### PR DESCRIPTION
**Pull request by dhakelila**
New styles for summary implemented. 

There is a minnor bug. When you change something, even if you back to the original state, you will never back to the initial state of the summary but you get a 0 instead. 
screen shot 2016-10-06 at 19 19 46

I'm gonna fix it.

### CODE REVIEW

**Comment by dhakelila** [comment]
Fixed

File: app/assets/javascripts/templates/analysis-total.hbs [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
+      <span class="title text -summary-small">Current practices</span>
+      <p class="value text -summary-big">{{total}}</p>
+    </div>
+    <div>
+      <span class="title text -summary-small">Potential future</span>
+      <p class="value text -summary-big">{{totalCompare}}</p>
+    </div>
+  </div>
+  <p class="text -analysis-total -bold">Introducing alternative practices would result in:</p>
+  <div class="susbtraction">
```
Comment (simaob) [inline app/assets/javascripts/templates/analysis-total.hbs]: typo in *subtraction*

File: app/assets/javascripts/templates/analysis-total.hbs [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
+    <p class="value text -figure-extra-big -highlighted">{{substraction}}</p>
```
Comment (simaob) [inline app/assets/javascripts/templates/analysis-total.hbs]: I'd suggest making the number green if it's negative and red if it's positive to show if the impact is positive or negative.

**Comment by simaob** [comment]
If you could address those two small things it's ready! =)

**Pull request merged**